e2e/process-services-cloud/apps-section-cloud.e2e.ts

```diff
@@ -30,15 +30,43 @@ describe('Applications list', () => {
     const appListCloudPage = new AppListCloudPage();
     const simpleApp = resources.ACTIVITI7_APPS.SIMPLE_APP;
 
+    const apiService = new ApiService('activiti', TestConfig.adf.hostBPM, TestConfig.adf.hostSso, 'BPM');
+    const identityService = new IdentityService(apiService);
+
+    let noRoleUser;
+
+    beforeAll( async () => {
+        await apiService.login(TestConfig.adf.adminEmail, TestConfig.adf.adminPassword);
+        const rolesService = new RolesService(apiService);
+
+        noRoleUser = await identityService.createIdentityUser();
+        const apsUserRoleId = await rolesService.getRoleIdByRoleName(CONSTANTS.ROLES.APS_USER);
+        await identityService.assignRole(noRoleUser.idIdentityService, apsUserRoleId, CONSTANTS.ROLES.APS_USER);
+    });
+
+    afterEach(async () => {
+        navigationBarPage.clickLogoutButton();
+        browser.executeScript('window.sessionStorage.clear();');
+        browser.executeScript('window.localStorage.clear();');
+    });
+
+    afterAll(async () => {
+        await identityService.deleteIdentityUser(noRoleUser.idIdentityService);
+    });
+
+    it('[C307058] Should be redirected to 403 when user doesn\'t have permissions', async () => {
```

Comment:
This test is about the AUTH guard SSO, please move it in a separate file related to the SSO. Plese give a look here https://github.com/Alfresco/alfresco-ng2-components/blob/development/docs/core/services/auth-guard-sso-role.service.md

Reply:
in general this test seems to test the same functionality of https://github.com/Alfresco/alfresco-ng2-components/pull/4612 so I guess you can remove it from here and testrail